src/daria/analysis/segmentationcomparison.py

```diff
@@ -0,0 +1,118 @@
+from __future__ import annotations
+
+import numpy as np
+
+
+def compare_segmentations(*segmentations, **kwargs) -> np.ndarray:
+    """
+    Comparison of segmentations.
+
+    Args:
+        segmentations (asterisk argument): Allows to provide an arbitraty
+                number of segmented images to be compared
+        optional keyword arguments (kwargs):
+            components (list): list of the different components that are
+                found in the segmented images. So far only two are allowed
+                to be provided.
+            gray_colors (np.ndarray): array of three different scales of
+                gray, one for each of the different combinations of
+                components in the segmentations.
+            colors (np.ndarray): Array of different colors that should
+                indicate uniqe components in each segmentation.
+            light_scaling (float): Indicate how much lighter the second
+                component should be scaled in its unique color.
+    """
+    # Define components
+    components: list = kwargs.pop("components", [1, 2])
+
+    # Define number of segmentations
+    N: int = len(segmentations)
```

Comment:
Use more descriptive variable name instead of single character variables for better search-functionality.

Reply:
Done